lib/config/executor/container_vm.go

```diff
@@ -34,7 +34,7 @@ type Common struct {
 	// A reference to the components hosting execution environment, if any
 	ExecutionEnvironment string
 
-	// Unambiguous ID with meaning in the context of its hosting execution environment
+	// Unambiguous ID with meaning in the context of its hosting execution environment. Change this definition will cause container backward compatibility issue. Please don't change this.
```

Comment:
Change/Changing

Also the comment might be little bit more than this so that reader can understand why he/she shouldn't be changing this